spec/index.html

```diff
@@ -408,6 +426,8 @@ <h2>A Canonical form of N-Quads</h2>
       <code>object</code>,
       and <code>graphLabel</code>,
       any of which MUST be a single <a href="#cp-space"><code title="space">space</code></a>.</li>
+     <li>A Canonical N-Quads document MUST NOT include a
+      <a href="#grammar-production-versionDirective"><code>VERSION</code></a> announcement directive.</li>
```

Comment:
`VERSION announcement directive` reads clumsily. Could this (and any similar instances) be changed to either `VERSION directive` or `VERSION announcement`?